kurtosis-module/impl/participant_network/prelaunch_data_generator/prelaunch_data_generator_launcher/prelaunch_data_generator_launcher.go

```diff
@@ -8,9 +8,8 @@ import (
 	"time"
 )
 
-
 const (
-	image = "skylenet/ethereum-genesis-generator:latest"
+	image = "ethpandaops/ethereum-genesis-generator:latest"
```

Comment:
Consequence of this one: https://github.com/ethpandaops/ethereum-genesis-generator/pull/12/files